src/main/java/seedu/address/storage/JsonAdaptedExam.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.storage;
```

Comment:
can confirm with isaac that there are no conflicts with his changes?

Reply:
yeah this exactly is the code i wrote